app/page.jsx

```diff
@@ -105,23 +114,33 @@ const Dashboard = () => {
           <div>
             {getUniqueGroups().map((group, groupIndex) => (
               <div key={groupIndex} className="mb-4">
-                <h2>{group}</h2>
-                <div className="lg:grid grid-cols-2 xl:grid-cols-4 gap-4">
-                  {filteredTickets
-                    .filter((ticket) =>
-                      viewBy === "category"
-                        ? ticket.category === group
-                        : ticket.status === group
-                    )
-                    .sort((a, b) => b.priority - a.priority)
-                    .map((filteredTicket, ticketIndex) => (
-                      <TicketCard
-                        id={ticketIndex}
-                        key={ticketIndex}
-                        ticket={filteredTicket}
-                      />
-                    ))}
+                <div
+                  className="flex justify-between items-center cursor-pointer bg-black-200 p-2 rounded"
+                  onClick={() => toggleGroupVisibility(group)}
+                >
+                  <h2 className="font-bold">{group}</h2>
+                  <span>
+                    {groupVisibility[group] ? "▼" : "►"}
```

Comment:
please avoid using the symbols directly.
```javascript
{groupVisibility[group] ? "▼" : "►"}
```

instead, please explore suitable FontAwesome icons for better consistency.